packages/app/src/client/components/NotificationsList/NotificationsList.tsx

```diff
@@ -40,21 +39,28 @@ export type NotificationsListProps = {
   }[];
 };
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(theme => ({
   item: {
     padding: 0,
-    display: 'block'
+    display: "block"
   }
 }));
 
-const NotificationsList = ({ isOpen, items, onClose, onDeleteAll }: NotificationsListProps) => {
+const NotificationsList = ({
+  isOpen,
+  items,
+  onClose,
+  onDeleteAll
+}: NotificationsListProps) => {
   const classes = useStyles();
 
-  const listNode = React.useRef<any>();
+  const listNode = React.useRef<HTMLUListElement>();
   const [height, setHeight] = React.useState<number>(0);
 
   useEffect(() => {
-    setHeight(listNode.current?.clientHeight + 45);
```

Comment:
Not quite sure if I understand this line.

if `listNode.current` is indeed `undefined` (as was anticipated with the `?` operator), it would evaluate to 
```typescript
undefined + 45
```
which is `NaN`